src/amperity/gocd/secret/vault/plugin.clj

```diff
@@ -218,14 +221,17 @@
   Params:
   - `client`: The vault.client (*not* as an Atom) you want to use to access Vault
   - `gocd-lookup-keys`: A seq of strings, (<PATH>#<KEY> ...), where <PATH> corresponds to a Vault Path, and <KEY>
-  a key found at that path."
-  [client gocd-lookup-keys]
+  a key found at that path.
+  - `force-read`: Ignore secret TTLs and read from Vault regardless of cached secrets?"
+  [client gocd-lookup-keys force-read]
   (let [;; A map from Vault paths (as keywords) to their corresponding secret data (defaults to nil)
         paths-to-vals
         (->> gocd-lookup-keys
              (map #(first (str/split % #"#")))
              (into #{})
-             (map (fn [path] [(keyword path) (vault/read-secret client path {:not-found nil})]))
+             (map (fn [path]
+                    [(keyword path) (vault/read-secret client path {:not-found nil
+                                                                    :force-read force-read})]))
```

Comment:
won't this be a string from configuration? I don't see when its parsed to a boolean.

Reply:
Good eye, fixed

Reply:
Fixing tests now